pytest_openfiles/plugin.py

```diff
@@ -97,14 +98,19 @@ def pytest_runtest_teardown(item, nextitem):
         ignore = False
 
         for ignored in open_files_ignore:
+
+            # Note that we use fnmatch rather than re for simplicity since
+            # we are dealing with file paths. In addition, we used to check
+            # for exact matches, and we need e.g. "astropy.log" to match
+            # only files with that exact name, and not e.g. "astropyxlog"
+            # which would happen if we switched to using regular expression.
+
             if not os.path.isabs(ignored):
-                if os.path.basename(filename) == ignored:
-                    ignore = True
-                    break
-            else:
-                if filename == ignored:
-                    ignore = True
-                    break
+                ignored = os.path.join('*', ignored)
```

Comment:
What is the intent of `os.path.join('*', ...)` here? Will there be unwanted side-effects injecting wildcard like this?

Reply:
I've added a comment about this - basically this is just because if the path is not absolute, it won't match the open file paths (which are absolute). So I'm just converting e.g. ``test.fits`` to ``*/test.fits`` so that it matches. Does that make sense?